src/components/SpriteMap/SpriteMap.vue

```diff
@@ -0,0 +1,297 @@
+<template>
```

Comment:
We should really generate this file. We've got scripts from other projects we can borrow.

Reply:
I would humbly ask to at least postpone it and do the merge. I'm sure this one is rock solid :)